Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
```

Comment:
### [method_source](http://banisterfiend.wordpress.com)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.8.2 → 0.9.2

[_~~change-log~~, [source-code](https://github.com/banister/method_source)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/banister/method_source/compare/v0.8.2...v0.9.2).

These are the first 10 commits:
- (3926a34) [Add Ruby 2 to Travis CI matrix. Eliminate warning.](https://github.com/banister/method_source/commit/3926a3405aad4b5c1dd76c35ef3a5e9c9bab4812)
- (5e5c556) [Merge pull request #24 from petergoldstein/feature/add_ruby_…](https://github.com/banister/method_source/commit/5e5c55642662c248e721282cc287b41a49778ee8)
- (8be4a97) [Add new Ruby 2.1 and 2.2 versions](https://github.com/banister/method_source/commit/8be4a97409fb91b20eab478bfc4a939757e73f62)
- (0c0a0ce) [Merge pull request #29 from nicolasleger/add_ruby_2_1_and_2_…](https://github.com/banister/method_source/commit/0c0a0cebbd11f2b44104b8c84b4f9952692422ad)
- (be10f27) [Add Travis CI status badge](https://github.com/banister/method_source/commit/be10f274ee02db4bd343ef29e2dc7ac4bf716e64)
- (05ce7fc) [Integer Unification for Ruby 2.4.0+](https://github.com/banister/method_source/commit/05ce7fcb22d3c7b0100d328a52376c6d5cb9272a)
- (3e2679d) [CI against Ruby 2.3](https://github.com/banister/method_source/commit/3e2679da63e4948fa87564c73700c6f4ab3264c0)
- (60e022e) [CI against Ruby 2.4.0](https://github.com/banister/method_source/commit/60e022edd866d19333f8825e87fdf5064ed73aee)
- (0e4ad87) [Merge pull request #39 from koic/integer_unification](https://github.com/banister/method_source/commit/0e4ad87b73794b271bffae4230382888207ed212)
- (6dcb116) [Merge pull request #40 from koic/use_latest_rubies](https://github.com/banister/method_source/commit/6dcb116e37e20e58f615ffe05a40bbe9a536e44a)


</details>